src/components/Message/Message.tsx

```diff
@@ -183,6 +186,11 @@ export const Message = <
   const handleRetry = useRetryHandler(propRetrySendMessage);
   const userRoles = useUserRole(message, onlySenderCanEdit, disableQuotedMessages);
 
+  const handleFetchReactions = useReactionsFetcher(message, {
+    getErrorNotification: getFetchReactionsErrorNotification,
+    notify: addNotification,
+  });
```

Comment:
This works similar to other handlers we define on the message context (see, for example, `handleMute`).